packages/app-store/_example/index.ts

```diff
@@ -0,0 +1,28 @@
+import type { App } from "@calcom/types/App";
+
+import _package from "./package.json";
+
+export const metadata = {
+  name: _package.name,
+  description: _package.description,
+  installed: true,
+  category: "video",
+  // If using static next public folder, can then be referenced from the base URL (/).
```

Comment:
Encouraging using imgSrc with / before next public path, otherwise you need to manually handle on every imgSrc={`/${props.logo}`}